packages/jest/test/printer.test.js

```diff
@@ -98,6 +98,33 @@ describe('jest-emotion with DOM elements disabled', () => {
   })
 })
 
+describe('jest-emotion with style insertion disabled', () => {
```

Comment:
could you flatten this to a single `it` block? the problem with putting `describe` in the middle is that now the "test report" won't match the order in the file and it's often confusing for me, since this doesn't group more than a single test i would prefer to just use a simple `it` here